README.md

```diff
@@ -80,8 +81,7 @@ The `connectionString` encapsulates the necessary information BTCPay needs to co
 * `type=lnbank;server=https://mybtcpay.com/lnbank;api-token=myapitoken`
 * `type=lndhub;server=https://login:password@lndhub.io`
 
-Note that `bitcoin-host` and `bitcoin-auth` are optional, only useful if you want to call `ILightningClient.GetDepositAddress` on Eclair.
```

Comment:
what happened to this?

Reply:
I thought this wasn't needed anymore. Checked, it still is, so I put it back and also added an example.